videostore/videostore.go

```diff
@@ -449,12 +449,12 @@ func (vs *videostore) fallbackDeleter(ctx context.Context) {
 	// This is a best effort to avoid running the fallback deleter immediately on startup to let the indexer
 	// run its cleanup first.
 	select {
-	case <-time.After(fallbackDeleterStartupDelay * time.Minute):
+	case <-time.After(fallbackDeleterStartupDelayMinutes * time.Minute):
```

Comment:
given this is the same interval, can't we remove this select statement?

Reply:
yeah good catch this code would wait 20 min before trying to do work on startup